src/lib/components/carrousel.svelte

```diff
@@ -117,38 +114,33 @@
     border: 2px solid var(--interactive-button);
     border-radius: 50%;
   }
-
   circle {
     fill: transparent;
     transition: fill 0.1s linear;
   }
   @media only screen and (max-width: 1024px) {
     .image {
-      aspect-ratio: 4/3;
+      aspect-ratio: 5/3;
+      padding-bottom: 0;
+      padding: 30px;
     }
-
     .carrousel {
-      display: flex;
-      justify-content: center;
-      aspect-ratio: 4/3;
-    }
-    ul {
-      bottom: 75px;
+      aspect-ratio: 5/3;
     }
-
     button.left,
     button.right {
       display: none;
     }
   }
   @media only screen and (max-width: 767px) {
-    ul {
-      bottom: 30px;
-    }
     .image {
```

Comment:
if this does not refer to img tag, then what does it refer to?

Reply:
is the div that collects the img

Reply:
rename